### ISSUE TITLE
unrecognise lib.rs in PyCharm

### ISSUE THREAD

**Issue by Turing-Chu** [intellij-rust/intellij-rust#2640]
I know that `intellij-rust` is developing for `intellij-idea`.  However, I found that `intellij-rust` also work in `PyCharm`.

but the `lib.rs` does not recognise as rust file but a regular file. like the following picture.
![image](https://user-images.githubusercontent.com/13805470/42305632-65a0933e-805e-11e8-8c83-e4b8ead77cf2.png)

**Comment by Undin** [comment]
@Turing-Chu 
> I know that intellij-rust is developing for intellij-idea. However, I found that intellij-rust also work in PyCharm.

Intellij-rust should work not only in IDEA because we almost don't have any IDE-specific features (except debugger) so it's expected that it works in PyCharm.

> but the lib.rs does not recognise as rust file but a regular file. like the following picture.

Could you provide your PyCharm and intellij-rust versions?

**Comment by Turing-Chu** [comment]
##### PyCharm
```
PyCharm 2018.1.3 (Professional Edition) Build #PY-181.4892.64, built on May 15, 2018
JRE: 1.8.0_152-release-1136-b38 x86_64
JVM: OpenJDK 64-Bit  Server VM by JetBrains s.r.o
```

##### intellij-rust
```
Version: 0.2.02099-181
Rust language support
Change Notes
https://intellij-rust.github.io/2018/07/04/changelog-77.html
Vendor
JetBrains
```

**Comment by Undin** [comment]
@Turing-Chu Sorry for so long response
Looks like that you have special type association for `lib.rs` file
You can check it in `Preferences > Editor > File Types`
If it's true, just remove this association

**Comment by Turing-Chu** [comment]
no. I'm not setting file type of rs file but setting file template in `Preferences > Editor > File and Code Templates`. Does it is this reason?

**Comment by Undin** [comment]
> setting file template in Preferences > Editor > File and Code Templates. Does it is this reason?

No, changes in file and code templates shouldn't change file type association

> no. I'm not setting file type of rs file

But maybe it happened by accident. Just type `lib.rs` in `Editor > File Type` preference window. It should show file type association that your PyCharm uses
<img width="1071" alt="screen shot 2018-07-13 at 22 02 24" src="https://user-images.githubusercontent.com/2539310/42709234-77724d42-86e8-11e8-8d56-0e9166f10b1d.png">

**Comment by Turing-Chu** [comment]
the same to you. I really don't know why it just does not recoganise lib.rs.
![image](https://user-images.githubusercontent.com/13805470/42721307-3e3562a2-876b-11e8-885b-8ac01b92bdb2.png)
